spec/models/user_spec.rb

```diff
@@ -15,6 +15,7 @@
   it { is_expected.to have_many(:user_providers).dependent(:destroy) }
   it { is_expected.to have_many(:flags).dependent(:destroy) }
   it { is_expected.to have_many(:notifications) }
+  it { is_expected.to have_many(:notifications).dependent(:destroy) }
```

Comment:
Is this supposed to be testing for likes?
```suggestion
  it { is_expected.to have_many(:likes).dependent(:destroy) }
```